src/bin.ts

```diff
@@ -6,6 +6,7 @@ import { Nuance } from './core/index.js'
 import { NuanceCLIOptions, NuanceConfigurationAllowedKeys } from './@types/index.js'
 import { onProcessExit } from './utils/before-exit.js'
 import { renderPageStatusList } from './view/cli.js'
+import { DirectusD } from './drivers/directus_driver.js'
```

Comment:
for consistency sake, you could export the driver as `DirectusDriver` instead of `DirectusD` (the more information the better), and rename the file as just `src/drivers/directus.ts` instead of `src/drivers/directus_driver.ts` (to remove redundancy)